### ISSUE TITLE
Hub01 Mission 2 Not Updating

### ISSUE THREAD

**Issue by TheMurderUnicorn** [CleverRaven/Cataclysm-DDA#41226]
# Describe the bug
Hub01, mission 2 where you mind splice the desiccated corpse is not working.  You can activate the splicer and it prints the message about transferring the data.  However, the Mission continues to point at the crash site, and returning to Hub01 does not let you complete the quest, it simply acts as though you haven't done it.

I tried on three separate worlds including a fresh one and the quest would not work on any of them.  Also spoke to someone who recently ran into the issue.  I know that's vague and not super helpful but it led me to believe it's not just a me issue.

# Steps To Reproduce

Debug reveal map
Find Hub01
Blah blah through mission 1.
Accept mission 2, Steal a Dead Man's Mind
Go to Crash Site and activate Mind Splicer on desiccated corpse
Observe printout about success.
See that Mission didn't update.
See that Intercom won't accept quest turn in.

# Expected behavior

Quest updates pointing back at hub to show you've completed the quest and are ready for turn in.
The intercom's dialogue should be the turn in dialogue.

# Screenshots

Here's a shot of the message log printout.  You'll see I received the success message and can even repeat it multiple times for some reason.  
![image](https://user-images.githubusercontent.com/41651793/84338372-d7e13d80-ab69-11ea-89fa-897c17e3a5d2.png)

# Versions and configuration
- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.E-3022-g28483f2 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    No Fungal Monsters [No_Fungi]
]

# Additional context
I confirmed with someone on Discord that I wasn't missing a step for the mission.  They also said it had been broken for them and they just assumed it was a random bug, not a continual issue.  For me it's happening on every World.

**Comment by John-Candlebury** [comment]
Did you receive a "Memory Card with Neural Data" after using the splicer? You should be able to turn that in.

Might be a container related bug

**Comment by TheMurderUnicorn** [comment]
No, I still have the regular data card and the mind splicer.  I believe that's the issue.  I originally thought it was the quest not updating, it appears that it does not produce the turn in item.  Someone also posted about this on the forum, where they stated it was an issue with being given that item.  https://discourse.cataclysmdda.org/t/bug-steal-a-dead-mans-mind/23919  Additionally, someone on Discord said there's an issue with another mission involving the blood drawing kit and that they believed it's the same issue causing it.

So, to summarize, I don't really know things but hopefully that's helpful somehow.

**Comment by IFailAtGaming** [comment]
can confirm it not producing the neural data, if spawned in through dev mode, quest progresses as normal

**Comment by pvgoddijn** [comment]
i currently cant turn in the data card with neural data, tried both having in in inventory and wielding it

**Comment by marningmol** [comment]
Same; hub 01 won't accept my "data card with neural data" that I got from using the mind splicer on the corpse.

**Comment by Melodeiro** [comment]
I can't even use mind splicer on corpse, it tells i don't have required item at hand. Memory card is in my inventory however.

**Comment by kholat** [comment]
The same thing happened to me (cant complete the quest with neural data in my inventory or wielded) so I continue my day and started a trade and interestingly enough after the trade both the mind splicer and the data card with neural data are gone, I did not trade them away theyre just gone; and the quest didnt complete.
